test/object-store/util/baas_admin_api.cpp

```diff
@@ -716,9 +722,16 @@ bool AdminAPISession::is_initial_sync_complete(const std::string& app_id) const
     return false;
 }
 
-AdminAPIEndpoint AdminAPISession::apps() const
+AdminAPIEndpoint AdminAPISession::apps(APIFamily family) const
 {
-    return AdminAPIEndpoint(util::format("%1/api/admin/v3.0/groups/%2/apps", m_base_url, m_group_id), m_access_token);
+    switch (family) {
+        case APIFamily::Admin:
+            return AdminAPIEndpoint(util::format("%1/api/admin/v3.0/groups/%2/apps", m_base_url, m_group_id),
+                                    m_access_token);
+        case APIFamily::Private:
+            return AdminAPIEndpoint(util::format("%1/api/private/v1.0/groups/%2/apps", m_base_url, m_group_id),
+                                    m_access_token);
+    }
```

Comment:
New warning on Jenkins linux CI:

baas_admin_api.cpp:735 | return-type | Normal | 1

control reaches end of non-void function [-Wreturn-type] 735